kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/FeedbackInputFragment.java

```diff
@@ -0,0 +1,196 @@
+package com.applozic.mobicomkit.uiwidgets.conversation.fragment;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import com.applozic.mobicomkit.uiwidgets.R;
+import com.applozic.mobicommons.commons.core.utils.Utils;
+
+/**
+ * fragment for the feedback input form
+ * @author shubham
+ * @date july 19
+ */
+public class FeedbackInputFragment extends Fragment {
+
+    private static final String TAG = "FeedbackInputFragment";
+
+    EditText editTextFeedback;
+    Button button1;
+    Button button2;
+    Button button3;
+    TextView textView1;
+    TextView textView2;
+    TextView textView3;
+    Button buttonSubmit;
+    Button closeButton;
+
+    FeedbackFragmentListener feedbackFragmentListener;
```

Comment:
Replace it with enum

Reply:
enums add unecessary overhead and increase the code size. If the constants checking is really required, I can use the intDef annotation.

Reply:
replaced with IntDef

Reply:
This performance patterns video by the Andorid Developers channel
https://www.youtube.com/watch?v=Hzs6OBcvNQE
Nd this article by chet hasse
https://medium.com/google-developers/developing-for-android-ii-bb9a51f8c8b9#.9q7762epb
Advice on not using enums was also given in the overhead section of the dev docs, but i cant seem to find it anymore.
However IntDef helps with similar constant checking.

Reply:
thanks, resolving this.